File: 8 kyu/MakeUpperCase/solution.py
def make_upper_case(s): return s.upper()
_____________________________________________
def make_upper_case(strng):
    return strng.upper()
_____________________________________________
make_upper_case = str.upper
_____________________________________________
def make_upper_case(s):
    return "".join(i.capitalize() for i in s)
_____________________________________________
def make_upper_case(s):
    print(s, s.upper())
    return s.upper()
_____________________________________________
def make_upper_case(s):
    new_string = ''
    for letter in s:
        new_string += letter.upper()
    return new_string
_____________________________________________
def make_upper_case(string):
    return string.upper()

make_upper_case("hello")
_____________________________________________
def make_upper_case(s):
    hola = s.upper()
    for palabra in s:
        return hola
_____________________________________________
def make_upper_case(s): return s.upper() #1 line bb

File: 8 kyu/MakeUpperCase/solution.rb
def make_upper_case(string)
  string.upcase
end
_____________________________________________
define_method :make_upper_case, &:upcase
_____________________________________________
def make_upper_case(str)
   upper = str.upcase
   upper
end
_____________________________________________
def make_upper_case(str) str.upcase end
_____________________________________________
def make_upper_case(str)
  str.bytes.map{|c| (97..122).include?(c) ? (c - 32).chr : c.chr}.join
end
_____________________________________________
def make_upper_case(str)
  # Code here
#   upcased = []
#   str.split.each do |letter|
#     if letter =~ (/^\s*$/)
#       upcased << letter
#     else
#       upcased << letter.upcase
#     end
#   end 
#   upcased.join("\s")
    str.upcase
end
